msgpack/unpack.scm

```diff
@@ -168,11 +171,16 @@
       (throw 'unexpected-eof))))
 
 
-(define (unpack-string size in)
-  "- Scheme Procedure: unpack-string size in
-     Unpack a UTF-8 string of length ‘size’ from binary input port ‘in’."
+(define (unpack-string-with-decoder size in proc)
+  "- Scheme Procedure: unpack-string-with-decoder size in proc
+     Unpack a string of length ‘size’ from binary input port ‘in’.
+
+    If proc is not #f, (‘proc’ bytevector) is called to decode the string,
+    otherwise it is decoded as utf8."
```

Comment:
Parameters are spelled in all-caps in the Guile manual, so I want to follow this convention.

```suggestion
     If PROC is not #f, ‘(PROC bytevector)’ is called to decode the string,
     otherwise it is decoded as UTF-8 ."
```